### PR TITLE
Added embed to help command.

### PR DESCRIPTION
**Pull request by gooca**

### CODE REVIEW

File: src/app/plugins/help.plugin.ts [hunk 1]
```diff
@@ -1,38 +1,39 @@
...
-  private _commandsResponse: string = '';
+  private _embed: IRichEmbed = new RichEmbed;
   constructor(public container: IContainer) {
     super();
   }
 
   public async execute(message: IMessage, args?: string[]) {
-    if (this._commandsResponse.length === 0) {
-      this._generateCommandsResponse();
+    if (this._embed.length === 0) {
```
Comment (gooca) [inline src/app/plugins/help.plugin.ts]: idk if checking the length of an object is supposed to work like correctly or with this syntax, but it works fine when I tested it. Not sure if this is what it should be doe

Comment (joey-colon) [reply src/app/plugins/help.plugin.ts]: You would probably want to be checking the length of the `fields` property which should contain an array of the fields within the RichEmbed object

File: src/app/plugins/help.plugin.ts [hunk 2]
```diff
@@ -1,38 +1,39 @@
...
-        this._commandsResponse += '\n';
-      }
-
-      this._commandsResponse += `**${plugin.name}**`;
-      this._commandsResponse += ` - `;
-      this._commandsResponse += `${plugin.description}`;
-      this._commandsResponse += ` - `;
-      this._commandsResponse += `\`${Constants.Prefix}${plugin.usage}\``;
+      this._embed
+      .addField(
```
Comment (joey-colon) [inline src/app/plugins/help.plugin.ts]: nit: indent

**Comment by joey-colon** [comment]
Resolves #33

**Comment by joey-colon** [comment]
lgtm 🐶

**Pull request merged**